userspace/libsinsp/cri.cpp

```diff
@@ -99,6 +99,17 @@ grpc::Status cri_interface::get_container_status(const std::string& container_id
 	return m_cri->ContainerStatus(&context, req, &resp);
 }
 
+grpc::Status cri_interface::get_container_stats(const std::string& container_id, runtime::v1alpha2::ContainerStatsResponse& resp)
```

Comment:
This doesn't look like it belongs in this commit (should be in the "CRIO support for rw.bytes.used" one)